install/index.md

```diff
@@ -275,7 +275,13 @@ This will drop you into a new shell where all software from the prefix is availa
 
 ROOT Docker containers for several linux flavours are available at [ROOT's official DockerHub](https://hub.docker.com/r/rootproject){:target="\_blank"}.
 
-For example, to try out the latest ROOT release just run `docker run -it rootproject/root`.
+For example, to try out the latest ROOT release just run the following command in your terminal (after starting docker engine):
+```cmd
+docker run --rm -it rootproject/root:latest
+```
+
+For more instructions on running root's docker image, visit [ROOT's official DockerHub](https://hub.docker.com/r/rootproject){:target="\_blank"}.
```

Comment:
That link sends me to https://hub.docker.com/u/rootproject

And that doesn't have any instructions. Maybe we can share generic Docker instructions? Or use https://hub.docker.com/r/rootproject/root as link?